src/NuGet.Clients/NuGet.CommandLine/Commands/InstallCommand.cs

```diff
@@ -234,8 +237,23 @@ private CommandLineSourceRepositoryProvider GetSourceRepositoryProvider()
 
             var allowPrerelease = Prerelease || (version != null && version.IsPrerelease);
 
+            DependencyBehavior depBehavior;
+            if(String.IsNullOrWhiteSpace(DependencyVersion))
+            {
+                // default behavior
+                depBehavior = DependencyBehavior.Lowest;
```

Comment:
@billboyletrintech I think this should read the value from the settings if it was not passed in.